modules/conversantBidAdapter.ts

```diff
@@ -23,18 +23,40 @@ import {ORTB_MTYPES} from '../libraries/ortbConverter/processors/mediaType.js';
  * @typedef {import('../src/adapters/bidderFactory.js').ServerRequest} ServerRequest
  * @typedef {import('../src/adapters/bidderFactory.js').Device} Device
  */
+const ENV = {
+  BIDDER_CODE: 'conversant',
+  SUPPORTED_MEDIA_TYPES: [BANNER, VIDEO, NATIVE],
+  ENDPOINT: 'https://web.hb.ad.cpe.dotomi.com/cvx/client/hb/ortb/25',
+  NET_REVENUE: true,
+  DEFAULT_CURRENCY: 'USD',
+  GVLID: 24
+} as const;
 
-const GVLID = 24;
+// parameters for the conversant adapter
+type ConversantBidParams = {
+  site_id: string;
+  tag_id: string;
+  bidfloor?: number;
+  secure?: boolean;
+  white_label_url?: string;
+  position?: number;
+  mimes?: string[];
+  maxduration?: number;
+  protocols?: number[];
+  api?: number[];
+}
```

Comment:
If you add documentation comments (`/** ...`) they'll be picked up by most tools, which is helpful for end users. I recommend repeating what you have your [doc](https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/epsilon.md), e.g.:

```
/**
 * If impression requires secure HTTPS URL creative assets and  markup. 0 for non-secure, 1 for secure. Default is non-secure
 */
secure?: boolean
```